### PR TITLE
Use correct npm version in CI

### PR DESCRIPTION
**Pull request by samwilson**

### CODE REVIEW

**Comment by samwilson** [comment]
I'm going to upgrade eslint.

**Comment by samwilson** [comment]
All passing now, this is ready for review. There isn't anything actually changed: just whitespace, eslint rule fixes, new i18n messages, etc.

File: public/assets/i18n/app/he.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	"preferred-langs-field-help": "השפות האלו תופענה בראש הרשימה בעת מעבר בין אחת לשנייה בעמוד התרגום.",
 	"add-preferred-lang": "הוספה",
 	"search-help": "נא להקליד שם קובץ מוויקישיתוף",
-	"search-label": "שם קובץ (ללא הקידומת ‚File:‎/קובץ:’) או כתובת:",
+	"search-label": "שם קובץ (ללא הקידומת ‚File:/קובץ:’) או כתובת:",
```
Comment (tstarling) [inline public/assets/i18n/app/he.json:27]: Looks like an LRM character was removed by eslint

Comment (samwilson) [reply public/assets/i18n/app/he.json:27]: Fixed. Also excluded all i18n files from eslint, because they're effectively linted by banana-checker.

**Pull request merged**